tools/webinstall/install.sh

```diff
@@ -1290,13 +1376,22 @@ main() {
     _main_auto_install="$_RETVAL"
     say_debug "Auto install: $_main_auto_install"
 
+    # Create kraftkit group and own files if it doesn't exist
+    check_environment "kraftkit" && create_environment "kraftkit"
+    _main_new_group="$?"
+
     # Install kraftkit for the given architecture
     install_kraftkit "$_main_arch" "$_main_auto_install"
     say "kraftkit was installed successfully to $PREFIX/kraft"
 
     # Check if kraft is installed and working
     kraft -h
 
+    # Check if we need to ask for restartqq
+    if [ "$_main_new_group" -eq 0 ]; then
+        say "please restart your shell to make kraftkit run commands available"
```

Comment:
https://unix.stackexchange.com/questions/18796/how-to-apply-changes-of-newly-added-user-groups-without-needing-to-reboot